buildSrc/src/main/groovy/bio.terra.policy.java-common-conventions.gradle

```diff
@@ -84,17 +84,11 @@ compileJava {
 
 // Spotbugs configuration
 spotbugs {
-    toolVersion = '4.7.3'
+    toolVersion = '4.9.3'
```

Comment:
update the spotbugs library version; this complements the `spotbugs-gradle-plugin` update